upload_artifacts.sh

```diff
@@ -1,21 +1,33 @@
 #!/bin/bash
 
-# Used in .travis.yml
+# This script is used in .travis.yml
+
+set -e
 
 BUCKET=piksi-buildroot-images
 PIKSI_VERSION=v3
+BUILD_VERSION=$(git rev-parse --short HEAD)
```

Comment:
Ahh, has no tags. Will this repo have tags? If it does, we should follow the semantics for version in other repos so that it makes things consistent (don't have to map between multiple version schemes).